contrib/xBOMEval/results/cdx1-pro/results-0804/linux.json

```diff
@@ -0,0 +1,1484 @@
+{
+  "questions": [
+    {
+      "question": "How do you list all files including hidden ones in the current directory?",
+      "answer": "Use the command 'ls -a'."
+    },
+    {
+      "question": "What command would you use to display the last 10 lines of a file named 'application.log'?",
+      "answer": "Use the command 'tail application.log'."
+    },
+    {
+      "question": "How can you count the number of lines in a text file using a single command?",
+      "answer": "Use the command 'wc -l filename'."
+    },
+    {
+      "question": "What is the difference between 'cp' and 'mv' commands in Linux?",
+      "answer": "'cp' copies files or directories, while 'mv' moves or renames them."
+    },
+    {
+      "question": "How do you create a new directory called 'project' and navigate into it in one command?",
+      "answer": "Use the command 'mkdir project && cd project'."
+    },
+    {
+      "question": "What command would you use to find all files with a '.txt' extension in the current directory and subdirectories?",
+      "answer": "Use the command 'find . -name \"*.txt\"'."
```

Comment:
Answer has unfortunately included `\"` to make things a valid json, although such escaping is not required for bash. zsh and fish require escaping for regex characters. Eg: `\*`